Jellyfin/Views/VideoViewModel.cs

```diff
@@ -70,12 +81,134 @@ public async void HandleParameters(Video.Parameters parameters)
             _playerElement.SetMediaPlayer(new MediaPlayer());
             _playerElement.MediaPlayer.Source = MediaSource.CreateFromAdaptiveMediaSource(ams);
             _playerElement.MediaPlayer.Play();
+            
 
             ams.InitialBitrate = ams.AvailableBitrates.Max();
+
+            _playerElement.MediaPlayer.MediaEnded += (mp, o) =>
+            {
+                _ = _jellyfinApiClient.UserPlayedItems[videoId].PostAsync();
```

Comment:
(unsure) Doesn't Jellyfin consider it played automatically if X% of the video is watched? I don't typically watch literally the entire thing for instance.